amun/entrypoint.py

```diff
@@ -30,23 +30,25 @@
 
 from thoth.common import datetime2datetime_str
 from thoth.common import init_logging
-from thoth.common import __version__ as thoth_common_version
+from thoth.storages import __version__ as __storages__version__
+from thoth.common import __version__ as __common__version__
 
 from datetime import datetime
 
 from . import __version__ as __amun_version__
 from .configuration import Configuration
 
 
-__version__ = f"{__amun_version__}+thoth_common.{thoth_common_version}"
-
-
 # Configure global application logging using Thoth's init_logging.
 init_logging(logging_env_var_start="AMUN_LOG_")
 
 _LOGGER = logging.getLogger("amun")
 _LOGGER.setLevel(logging.DEBUG if bool(int(os.getenv("AMUN_DEBUG", 0))) else logging.INFO)
 
+__service_version__ = f"{__amun_version__}+storage.{__storages__version__}.common.{__common__version__}"
```

Comment:
@fridex Should I name it just `__version__` instead of `__service_version__`? I also fixes some `pre-commit` fails, could you PTAL?

Reply:
`__service_version__` is correct, the `__version__` in `__init__.py` will be used by bots.